os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -240,8 +240,10 @@ void board_gpio_initialize(void)
 		u32 pinpull;
 	} pins[] = {
 				{PA_23, PIN_INPUT, PullNone},
+				{PA_4, PIN_INPUT, PullNone},
```

Comment:
Please remove it in here.

we have no plan to register gpio driver about this pin.